plugins/backgroundrunner.koplugin/luawrapper.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Converts the return of "sh wrapper.sh $@" into Lua format.
+
+CURRENT_DIR=`dirname "$0"`
```

Comment:
See the automated tests for the shellcheck and formatting issues (e.g., on this line use `$(dirname "$0")`, use four spaces to abide by the coding style, etc.)